app/main.py

```diff
@@ -2,0 +1,21 @@
+def copy_file(command: str) -> None:
+    commands = command.split()
+    if (
+        len(commands) < 3
+        or commands[0] != "cp"
+        or commands[1] == commands[2]
+    ):
+        return
+    try:
+        with (
+            open(commands[1], "r") as source,
+            open(commands[2], "a") as target
+        ):
+            for line in source:
+                target.write(line)
```

Comment:
Code Style: The variable names `source` and `target` are descriptive and well-chosen, which is good practice. However, you should use double quotes consistently throughout your code, as per the checklist.